### ISSUE TITLE
Facing following issues building latest versions

### ISSUE THREAD

**Issue by swaruprs** [realm/realm-java#5267]
classpath "io.realm:realm-gradle-plugin:3.8.0"

    Error:Could not find io.realm:realm-gradle-plugin:3.8.0.
    Searched in the following locations:
    file:/Applications/Android Studio.app/Contents/gradle/m2repository/io/realm/realm-gradle-plugin/3.8.0/realm-gradle-plugin-3.8.0.pom
    file:/Applications/Android Studio.app/Contents/gradle/m2repository/io/realm/realm-gradle-plugin/3.8.0/realm-gradle-plugin-3.8.0.jar
    https://jcenter.bintray.com/io/realm/realm-gradle-plugin/3.8.0/realm-gradle-plugin-3.8.0.pom
    https://jcenter.bintray.com/io/realm/realm-gradle-plugin/3.8.0/realm-gradle-plugin-3.8.0.jar
    Required by:
    project :

classpath "io.realm:realm-gradle-plugin:4.0.0-BETA1-91d84de"

    Error:Could not find io.realm:realm-gradle-plugin:4.0.0-BETA1-91d84de.
    Searched in the following locations:
    file:/Applications/Android Studio.app/Contents/gradle/m2repository/io/realm/realm-gradle-plugin/4.0.0-BETA1-91d84de/realm-gradle-plugin-4.0.0-BETA1-91d84de.pom
    file:/Applications/Android Studio.app/Contents/gradle/m2repository/io/realm/realm-gradle-plugin/4.0.0-BETA1-91d84de/realm-gradle-plugin-4.0.0-BETA1-91d84de.jar
    https://jcenter.bintray.com/io/realm/realm-gradle-plugin/4.0.0-BETA1-91d84de/realm-gradle-plugin-4.0.0-BETA1-91d84de.pom
    https://jcenter.bintray.com/io/realm/realm-gradle-plugin/4.0.0-BETA1-91d84de/realm-gradle-plugin-4.0.0-BETA1-91d84de.jar
    Required by:
    project :

Even versions

    4.0.0-BETA2-afe7114
    4.0.0-BETA3-SNAPSHOT

However,

classpath "io.realm:realm-gradle-plugin:3.7.0" 

works good

**Comment by Zhuinden** [comment]
Have you added

```
buildscript {
    repositories {
        maven {
            url 'http://oss.jfrog.org/artifactory/oss-snapshot-local'
        }
    }

```
For snapshots?

**Comment by swaruprs** [comment]
// Top-level build file where you can add configuration options common to all sub-projects/modules.

    buildscript {
     repositories {
        jcenter()

        maven {
            url 'http://oss.jfrog.org/artifactory/oss-snapshot-local'
        }
     }
     dependencies {
        classpath 'com.android.tools.build:gradle:2.3.3'
        // retrolambda
        classpath 'me.tatarka:gradle-retrolambda:3.7.0'
        // realm install
        classpath "io.realm:realm-gradle-plugin:4.0.0-BETA2-afe7114-SNAPSHOT" /*3.1.1*/
        classpath 'com.google.gms:google-services:3.1.0'
     }
    }

    allprojects {
     repositories {

        mavenCentral()

        jcenter()
        // Stetho by maven
        maven {
            url 'https://github.com/uPhyca/stetho-realm/raw/master/maven-repo'
        }
        // realm database browser
        maven { url "https://jitpack.io" }
     }
    }

    task clean(type: Delete) {
     delete rootProject.buildDir
    }

**Error**: Failed to resolve: io.realm:realm-android-library:4.0.0-BETA2-afe7114-SNAPSHOT

**Comment by Zhuinden** [comment]
I would try the name `4.0.0-BETA3-SNAPSHOT`

**Comment by swaruprs** [comment]
Error:Could not find io.realm:realm-gradle-plugin:4.0.0-BETA3-SNAPSHOT.
Searched in the following locations:
    file:/Applications/Android Studio.app/Contents/gradle/m2repository/io/realm/realm-gradle-plugin/4.0.0-BETA3-SNAPSHOT/maven-metadata.xml
    file:/Applications/Android Studio.app/Contents/gradle/m2repository/io/realm/realm-gradle-plugin/4.0.0-BETA3-SNAPSHOT/realm-gradle-plugin-4.0.0-BETA3-SNAPSHOT.pom
    file:/Applications/Android Studio.app/Contents/gradle/m2repository/io/realm/realm-gradle-plugin/4.0.0-BETA3-SNAPSHOT/realm-gradle-plugin-4.0.0-BETA3-SNAPSHOT.jar
    https://jcenter.bintray.com/io/realm/realm-gradle-plugin/4.0.0-BETA3-SNAPSHOT/maven-metadata.xml
    https://jcenter.bintray.com/io/realm/realm-gradle-plugin/4.0.0-BETA3-SNAPSHOT/realm-gradle-plugin-4.0.0-BETA3-SNAPSHOT.pom
    https://jcenter.bintray.com/io/realm/realm-gradle-plugin/4.0.0-BETA3-SNAPSHOT/realm-gradle-plugin-4.0.0-BETA3-SNAPSHOT.jar
    http://oss.jfrog.org/artifactory/oss-snapshot-local/io/realm/realm-gradle-plugin/4.0.0-BETA3-SNAPSHOT/maven-metadata.xml
    http://oss.jfrog.org/artifactory/oss-snapshot-local/io/realm/realm-gradle-plugin/4.0.0-BETA3-SNAPSHOT/realm-gradle-plugin-4.0.0-BETA3-SNAPSHOT.pom
    http://oss.jfrog.org/artifactory/oss-snapshot-local/io/realm/realm-gradle-plugin/4.0.0-BETA3-SNAPSHOT/realm-gradle-plugin-4.0.0-BETA3-SNAPSHOT.jar
Required by:
    project :

**Comment by Zhuinden** [comment]
Hmm, apparently you're supposed to find it by the name `4.0.0-BETA2-afe7114-SNAPSHOT`.

That's odd.

**Comment by swaruprs** [comment]
Ahh,

Error:Could not find io.realm:realm-gradle-plugin:4.0.0-BETA3-SNAPSHOT

after I tried

classpath "io.realm:realm-gradle-plugin:4.0.0-BETA3-SNAPSHOT"

**Comment by cmelchior** [comment]
We had some issues publishing our SNAPSHOT. Those should now be resolved, so `4.0.0-BETA3-SNAPSHOT` should work now.
